AUTHORS

```diff
@@ -1,11 +1,15 @@
-Pyccel is written and maintained by Ratnani Ahmed <ratnaniahmed@gmail.com>.
-
-Other co-maintainers:
-
-*
-
-Other contributors, listed alphabetically, are:
-
-*
-
-Many thanks for all contributions!
+Maintainers
+-----------
+* Ahmed Ratnani <ratnaniahmed@gmail.com> (project creator, owner)
+* Yaman Güçlü <yaman.guclu@gmail.com> (administrator, owner)
+* Said Hadjout
+* Emily Bourne
+
+Contributors
+------------
+* Karim Daou
+* Mouad Elalj
+* Mostafa Mamouni
+* Noureddine Hamid
+* Nouredine Ouhaddou
+* Redone Mahjoubi
```

Comment:
@yguclu according to github (https://github.com/pyccel/pyccel/graphs/contributors)
Chris has also contributed as well as user toddrme2178 (I am not sure who that is). Should they be added to the contributors?

Reply:
I was thinking of this as a list of people that have made substantial contributions to Pyccel. In the future we could think of having two separate lists, one for "current developers" and one for "past contributors", but maybe we should take care of this in the RELEASE file that should be attached to each new Pyccel version.

Potentially we could have another file of thanks/acknowledgments where we list every person who contributes even just 2 lines to Pyccel, but that seems a bit overkill to me right now.

@krystophny Would you like to be listed among the pyccel contributors in this file?

Reply:
@yguclu It would be an honor!

Reply:
@mostafaMamoni Please add "Christopher Albert" to this list!

Reply:
yes sir

Reply:
Looking at the commit message "toddrme2178 authored and ratnania committed", I think toddrme2178 might be Ahmed